analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/CreatingHashAlgorithms.cs

```diff
@@ -62,6 +62,10 @@ public void Hash(byte[] temp)
             var md5Var = (HashAlgorithm)CryptoConfig.CreateFromName(algoName); // Noncompliant
             algoName = "SHA256Managed";
             var SHA256ManagedVar = (HashAlgorithm)CryptoConfig.CreateFromName(algoName);
+
+            const string part1 = "System.Security.Cryptography";
```

Comment:
Please also add a test case that documents that non-constant interpolated string values are not evaluated.